app/services/proofing/lexis_nexis/ddp/verification_request.rb

```diff
@@ -0,0 +1,45 @@
+module Proofing
+  module LexisNexis
+    module Ddp
+      class VerificationRequest < Request
+        private
+
+        def build_request_body
+          {
+            api_key: config.api_key,
+            org_id: config.org_id,
+            account_address_street1: applicant[:address1],
+            account_address_street2: applicant[:address2] || '',
+            account_address_city: applicant[:city],
+            account_address_state: applicant[:state],
+            account_address_country: 'US',
+            account_address_zip: applicant[:zipcode].match(/^\d{5}/).to_s,
+            account_date_of_birth: Date.parse(applicant[:dob]).strftime('%Y%m%d'),
+            account_email: applicant[:email],
+            account_first_name: applicant[:first_name],
+            account_last_name: applicant[:last_name],
+            account_telephone: applicant[:phone],
+            drivers_license_number_hash: applicant[:state_id_number] ?
+              OpenSSL::Digest::SHA256.hexdigest(applicant[:state_id_number].gsub(/\D/, '')) : '',
```

Comment:
I'm not sure about other states/territories, but at least in Wisconsin the ID number includes a letter, which would be removed by the regex, is that ok?

Reply:
yeah maybe we just want to strip space characters? leaving in letters is important in California too

Reply:
ah great catch.  my license begins with a letter...

Reply:
I'll confirm with LN  that "Hash of the Drivers Licence Number" is not verbatim ie purely the number.  their docs also misspelled license! lol